docs/tech-preview-blog-post.md

```diff
@@ -0,0 +1,128 @@
+# Blog Post for Technical Preview
+
+Unicode announced today the Technical Preview of MessageFormat 2,
+a new standard for creating and managing user interface strings.
+These messages can dynamically include data values formatted
```

Comment:
This goes from "strings" to "messages" and it's not clear that you're talking about the same thing.